### PR TITLE
Db migration setup with models and refrences

### PR DESCRIPTION
**Pull request by amansson**

### CODE REVIEW

File: app/models/pdf.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Pdf < ApplicationRecord
+  belongs_to :product_medium, as: :mediable
```
Comment (Pidd69) [inline app/models/pdf.rb]: REMOVE dirtbag!

File: app/models/video.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Video < ApplicationRecord
+  belongs_to :product_medium, as: :mediable
```
Comment (Pidd69) [inline app/models/video.rb]: Remove monkeypants!

**Review by Pidd69 — CHANGES_REQUESTED** [review]
Remove :mediable on class Video and pdf

File: app/models/video.rb (hunk 2 above)
Comment (amansson) [reply app/models/video.rb]: Fixed

**Review by Pidd69 — APPROVED** [review]

**Pull request merged**